.github/workflows/next.js.yml

```diff
@@ -24,4 +24,5 @@ jobs:
     - run: yarn install --frozen-lockfile
     - run: yarn build
     - run: yarn test
+    - run: yarn lint --quiet
```

Comment:
are there useless warnings? just curious why --quiet

Reply:
hmm, good question. I was following the suggested set up where you have also done the commit hooks, so I think the idea is this should very rarely yell unless your developer sidestepped the commit hook? but happy to let it be verbose

Reply:
gotcha, no opinion then